tests/Controllers/PostControllerTest.php

```diff
@@ -52,11 +52,13 @@ public function testCanUserCreateAPost()
         ]);
 
         $fileUrl  = base_url('uploads/' . $user->id . '/test_image1.jpg');
-        $response = $this->actingAs($user)->post('posts/1', [
-            'thread_id' => '1',
-            'reply_to'  => '',
-            'body'      => 'Sample body for post ![](' . $fileUrl . ')',
-        ]);
+        $response = $this
+            ->withHeaders([csrf_header() => csrf_hash()])
```

Comment:
We should consider an addition either here or in CI itself that simplfies this, like `->withCSRF(()` at some point...

Reply:
I take that back. This isn't something the devs should have to think about at all, which means it's definitely something for the core, I think.

Reply:
Yeah, not sure, honestly. Will be a bit before I can dig into the core code to see what's going on. Trying to wrap up the security page tonight. I probably should have broken that up into a few different PRs, but too late now :) 

To be fair, I don't recall ever having to do that manually in the past. If we do, it's definitely a step backwards and potential blocker to getting our community more into testing.